docs/source/home.rst

```diff
@@ -2,7 +2,7 @@
 Home
 ####
 
-Vircadia is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
+Vircadia™ is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
```

Comment:
This seems to be up for debate. The (TM) is supposedly not required by law and only makes it sound like we are going to be asshats about people using the name.

Reply:
It's mainly to be safe. I have to grant permission to use the name in order to protect the trademark for the project. Giving notice helps with confusion as we've also seen as well. lol